libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -403,6 +406,7 @@ class AC_PosControl
     float       _leash;                 // horizontal leash length in cm.  target will never be further than this distance from the vehicle
     float       _leash_down_z;          // vertical leash down in cm.  target will never be further than this distance below the vehicle
     float       _leash_up_z;            // vertical leash up in cm.  target will never be further than this distance above the vehicle
+    float       _vel_z_control_ratio = 2.0f;   // confidence that we have control in the vertical axis
```

Comment:
maybe extend the comment to add (0 means no confidence, above 1 means full confidence)"